src/psyclone/psyir/nodes/structure_reference.py

```diff
@@ -160,7 +188,7 @@ def _create(cls, symbol, symbol_type, members, parent=None):
                 f"symbol '{symbol.name}'")
 
         # Create the base reference to the symbol that is a structure
-        ref = cls(symbol, parent=parent)
+        ref = cls(symbol=symbol, parent=parent)
```

Comment:
You could revert this named argument addition if you solve the positional argument issue I mentioned above.

(although I don't really mind how it is called here)

Reply:
Done.